src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml

```diff
@@ -193,7 +193,7 @@
 
 @section breadcrumb {
     <div class="breadcrumbs">
-        <a href="@Url.Action("ListAll", "EmployerManageApprentices")" aria-label="Back to manage your apprentices" class="back-link">Back to manage your apprentices</a>
+        <a href="@Url.Action("ListAll", "EmployerManageApprentices", new {checkCookie=true})" aria-label="Back to manage your apprentices" class="back-link">Back to manage your apprentices</a>
```

Comment:
is checkCookie always going to be true?

Reply:
yep, always true from that link. its the only time to check for the cookie.